etc/sds_operations_ceph-release-1-0.yaml

```diff
@@ -1,436 +1,108 @@
+namespace.tendrl.ceph_integration:
+  object_details:
+    Tendrl_context:
+      enabled: True
+      attrs:
+        cluster_id:
+          help: "Tendrl managed/generated cluster id for the sds being managed by Tendrl"
+          type: String
+        sds_name:
+          help: "Name of the Tendrl managed sds, eg: 'gluster'"
+          type: String
+        sds_version:
+          help: "Version of the Tendrl managed sds, eg: '3.2.1'"
+          type: String
+        node_id:
+          help: "Tendrl ID for the managed node"
+          type: String
+    Pool:
+      enabled: True
+      value: /clusters/$cluster.id/pools/$pool.id
+      attrs:
+        poolname:
+          type: String
+          help: "Name of the Ceph pool"
+        pooltype:
+          type: String
+          help: "Type of the Ceph pool(ec or replicated)"
+        replica_count:
+          type: Integer
+          help: "Replica count of volume"
+        pg_num:
+          type: Integer
+          help: "The total number of placement groups for placement purposes."
+        pgp_num:
+          type: Integer
+          help: "The total number of placement groups for the pool."
+        crush_ruleset:
+          type: Integer
+          help: "The ID of a CRUSH ruleset to use for this pool. The specified ruleset must exist."
+        ec_profile:
+          type: String
+          help: "For erasure pools only.It must be an existing profile "
+        max_objects:
+          type: Integer
+          help: "quota for maximum number of objects. To remove quota set the value to zero"
+        max_bytes:
+          type: Integer
+          help: "quota for maximum number of bytes. To remove quota set the value to zero"
+    atoms:
+        create:
+          enabled: True
+          inputs:
+            mandatory:
+              - Pool.poolname
+              - Pool.pooltype
+              - Pool.replica_count
+              - Pool.pg_num
+              - Pool.pgp_num
+              - Pool.crush_ruleset
+            optional:
+              - Pool.max_objects
+              - Pool.max_bytes
+              - Pool.ec_profile
+          name: "Create Pool"
+          run: tendrl.ceph_integration.objects.Config.atoms.create_pool
+          type: Create
+          uuid: bd0155a8-ff15-42ff-9c76-5176f53c13e0
+        delete:
+          enabled: True
+          inputs:
+            mandatory:
+              - Pool.poolname
+          name: "Delete Pool"
+          run: tendrl.ceph_integration.objects.Config.atoms.delete_pool
+          type: Delete
+          uuid: 9a2df258-9b24-4fd3-a66f-ee346e2e3720
+    flows:
+      CreatePool:
+        atoms:
+          - tendrl.ceph_integration.objects.Config.atoms.create_pool
+        description: "Create Ceph Pool"
+        enabled: True
+        inputs:
+            mandatory:
+              - "Pool[]"
+              - Tendrl_context.sds_name
+              - Tendrl_context.sds_version
+              - Tendrl_context.cluster_id
```

Comment:
how does the create pool atom get other inputs like pool name, pool type , replica count etc ?